content/glossary.md

```diff
@@ -3,19 +3,52 @@ title: Glossary
 kind: Documentation
 draft: true
 ---
+
 **Agent**
 
+**API**
+
+Datadog provides an HTTP API in order to interact programmatically with the platform. Every feature, resource, mechanism, and bit of functionality can be accessed via the API. Interacting with the API manually is a great way to learn how Datadog works under the hood! In production however, you will use a purpose-built tool, library, or interface — such as the Datadog product itself — as an abstraction layer between you and the API.
+
+See the [API documentation][1] for more information.
+
 **Check**
 
+Checks are small Python programs run periodically by the Agent. A Check performs an action and then gathers the result, which the Agent then stores and reports to the Datadog platform. These programs are freeform and are generally used to collect metrics from custom environments or applications.
```

Comment:
periodically is a bit vague. what is the period, 10-15 seconds. Periodically could mean every few hours

Reply:
'A Check...' : does check need to be capitalized? if refering to the word being defined, maybe **bolding it is better**

Reply:
We don't want to get into the weeds in terms of details. The actual period is an implementation detail that's covered in the in-depth documentation.

Check, as in the abstraction describing the thing the Agent does, is a proper name in the Datadog glossary. This "fact" causes weird writing problems and I think we should re-visit it in any case.